app/views/layouts/_head.html.erb

```diff
@@ -36,6 +36,17 @@
   <%= include_account_css %>
   <script>
     window.bz_custom_css_url = <%= custom_css_url.to_json.html_safe %>;
+    try {
+      if(window.location.pathname != '/' && window.location.pathname.indexOf('/login') != 0) {
```

Comment:
On Thu, Jul 02, 2015 at 08:47:45AM -0700, Brian Sadler wrote:
> This seems brittle.  Are there no built-in variable names for root and login routes?

This covers ALL the routes on the login thing (notice it
is indexOf rather than plain equality) - /login, /login/canvas,
/login/cas, the list goes on. Checking all of them would be
a lot of variables and could break if a new one is added...
the other reasonable alternative might be checking if there is
a current_user right now, since they'd be logged out on the
login page it might be reasonable to use that instead.


The root could use root_path but meh, the root is by definition
going to be '/' anyway...